### PR TITLE
Search: update docs for before/after to mention UTC

### PR DESCRIPTION
**Pull request by camdencheek**
I skipped the tooltip hovers because we need those to be short. 

## Test plan

N/A

## App preview:

- [Web](https://sg-web-cc-before-after-docs.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.08 kb) | 0.00% (+0.08 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/cc/before-after-docs/compare-report.html;tab=live_object) for a full comparison between the commits c881ec5 and 49b2aa4 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Review by limitedmage — APPROVED** [review]

**Comment by limitedmage** [comment]
Can we also update the tip here? 
image

File: client/search-ui/src/results/sidebar/SearchReference.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const filterInfos: FilterInfo[] = [
         ...createQueryExampleFromString('"{last week}"'),
         field: FilterType.after,
         description:
-            'Only include results from diffs or commits which have a commit date after the specified time frame. To use this filter, the search query must contain `type:diff` or `type:commit`.',
+            'Only include results from diffs or commits which have a commit date after the specified time frame. Times are interpreted as UTC by default. To use this filter, the search query must contain `type:diff` or `type:commit`.',
```
[the remark this answers is not in the document]
Comment (camdencheek) [reply client/search-ui/src/results/sidebar/SearchReference.tsx:81]: We [do support](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@71cb15117b34d2f6444352e8bf183f642e776ce0/-/blob/internal/search/query/date_format.go?L40-43) dates with timezones, so it's only when the timezone is unspecified do we assume UTC.

[the remark this answers is not in the document]
Comment (limitedmage) [reply client/search-ui/src/results/sidebar/SearchReference.tsx:81]: Ohh got it! Nice!

**Pull request merged**